docs/src/pages/landing-page/Index.vue

```diff
@@ -29,13 +29,18 @@
       />
 
       <div class="intro-section__sponsors-heading q-mt-xl text-weight-bold text-lp-primary text-size-16 text-capitalize">Our Platinum sponsors</div>
-      <q-img
-        v-for="(src, index) in sponsorLogos.platinum"
-        :key="index"
-        :src="`sponsor-logos/${src}`"
-        width="200px"
-        class="q-my-md"
-      />
+      <a
```

Comment:
Can we create a "SponsorLink" component and use it every time a sponsor logo is shown?